src/utils/constants.js

```diff
@@ -0,0 +1,2 @@
+export const todosURL = 'https://jsonplaceholder.typicode.com/todos';
+export const usersURL = 'https://jsonplaceholder.typicode.com/users';
```

Comment:
зачем нам две одинаковые ссылки
```suggestion
export const API_URL = 'https://jsonplaceholder.typicode.com/';
```

в нужном месте дописывай нужные параметры (`todos / users`)